staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/forklift/v1beta1/types.go

```diff
@@ -0,0 +1,86 @@
+package v1beta1
+
+import (
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OvirtVolumePopulatorKind is the type of the CR used to populator a volume from an oVirt disk
+var OvirtVolumePopulatorKind = "OvirtVolumePopulator"
+
+// OvirtVolumePopulator is the CR used to populator a volume from an oVirt disk
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+// +kubebuilder:resource:shortName={ovvp,ovvps}
+type OvirtVolumePopulator struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec OvirtVolumePopulatorSpec `json:"spec"`
+	// +optional
+	Status OvirtVolumePopulatorStatus `json:"status"`
+}
+
+// OvirtVolumePopulatorSpec is the spec of the OvirtVolumePopulator CR
+type OvirtVolumePopulatorSpec struct {
+	EngineURL        string `json:"engineUrl"`
+	EngineSecretName string `json:"engineSecretName"`
+	DiskID           string `json:"diskId"`
+	// The network attachment definition that should be used for disk transfer.
+	TransferNetwork *core.ObjectReference `json:"transferNetwork,omitempty"`
```

Comment:
Are we consistent with kubevirt/kubevirt wrt specifying the network here?
If that's a NetworkAttachmentDefinition, I believe kubevirt uses plain string nad_name?

Reply:
forklift used the term transfer network, so that's the reason for the name, I have no problem changing it to nad_name

Reply:
the name is fine I meant the type

Reply:
ah ok, yeah it's the same type used in https://github.com/kubev2v/forklift/blob/main/pkg/apis/forklift/v1beta1/plan.go#L44

anyway, it's good that you've mentioned it because I realize I don't actually use it in the controller, which I should

Reply:
I meant if it's consistent with kubevirt - https://kubevirt.io/user-guide/virtual_machines/interfaces_and_networks/#multus